storage-label-videos/README.md

```diff
@@ -0,0 +1,64 @@
+# storage-label-videos
+
+**Author**: Firebase (**[https://firebase.google.com](https://firebase.google.com)**)
+
+**Description**: Extracts labels from your videos uploaded to Storage and writes the extracted labels to Storage as a JSON file.
+
+---
+
+## 🧩 Install this experimental extension
+
+> ⚠️ **Experimental**: This extension is available for testing as an _experimental_ release. It has not been as thoroughly tested as the officially released extensions, and future updates might introduce breaking changes. If you use this extension, please [report bugs and make feature requests](https://github.com/firebase/experimental-extensions/issues/new/choose) in our GitHub repository.
+
+### Console
+
+[![Install this extension in your Firebase project](../install-extension.png?raw=true "Install this extension in your Firebase project")](https://console.firebase.google.com/project/_/extensions/install?sourceName=)
+
+### Firebase CLI
+
+```bash
+firebase ext:install storage-label-videos --project=<your-project-id>
+```
+
+> Learn more about installing extensions in the Firebase Extensions documentation: [console](https://firebase.google.com/docs/extensions/install-extensions?platform=console), [CLI](https://firebase.google.com/docs/extensions/install-extensions?platform=cli)
+
+---
+
+**Details**: TODO
```

Comment:
TODO

Reply:
@dackers86 if you regenerate the readme (add that early return that I showed you) it will update these for you based on the extension.yaml

Reply:
Thanks. All updated